.travis.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       compiler: clang
       env: >
         CLANG_SANITIZER=ASAN_UBSAN
-        CMAKE_FLAGS="$CMAKE_FLAGS -DPREFER_LUA=ON"
+        CMAKE_FLAGS="$CMAKE_FLAGS -DPREFER_LUA=OFF"
```

Comment:
This is intentionally using Lua (c.f. 8fec4d53d0) because we build Lua with ASAN to be able to test our embedded Lua support.